### PR TITLE
feat: Allow enabling payment callbacks when creating an order

### PR DESCRIPTION
**Pull request by frankverhoeven**
Not every payment status change results in an order status change; thus a
relevant payment status change may not be provided via the order webhook
callback.
With this change it will be (optionally) possible to enable callbacks for
payment status changes in addition to order status changes. Moreover, it
is possible to embed the payments in the response when creating an order.
Payment status callbacks will provide a payment ID, thus this information
has to be saved along with the order ID.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/.gitattributes export-ignore
```
Comment (kusozayka) [inline .gitattributes:1]: what's wrong with using `.gitignore`?

Comment (frankverhoeven) [reply .gitattributes:1]: This file serves a different purpose: https://git-scm.com/docs/gitattributes#_export_ignore
TL;DR: It allows you to exclude files from the archive, thus excluding them when installing the package via composer.
